ee/localserver/krypto-ec-middleware.go

```diff
@@ -216,6 +218,41 @@ func (e *kryptoEcMiddleware) Wrap(next http.Handler) http.Handler {
 			}()
 		}
 
+		// Check if the origin is in the allowed list. See https://github.com/kolide/k2/issues/9634
+		if len(cmdReq.AllowedOrigins) > 0 {
```

Comment:
I think this needs to move above the previous block, unless I'm misunderstanding what we're trying to accomplish -- we kick off `e.sendCallback` on 217 before this check, so the callback will still execute even if the origin isn't allowed.

Reply:
I missed the `defer` above + the fact that we want to send callbacks in the error case too